src/applications/hca/tests/config/serviceInformation.unit.spec.js

```diff
@@ -1,88 +1,89 @@
   const { schema, uiSchema } = formConfig.chapters.militaryService.pages.serviceInformation;
-  const definitions = formConfig.defaultDefinitions;
-  it('should render', () => {
-    const form = ReactTestUtils.renderIntoDocument(
+
+  it('renders military info', () => {
+    const form = mount(
       <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
         schema={schema}
         data={{}}
-        uiSchema={uiSchema}
-        definitions={definitions}/>
+        formData={{}}
+        uiSchema={uiSchema}/>
     );
-    const formDOM = getFormDOM(form);
 
-    expect(formDOM.querySelectorAll('input').length).to.equal(2);
-    expect(formDOM.querySelectorAll('select').length).to.equal(6);
+    expect(form.find('input').length).to.equal(2);
+    expect(form.find('select').length).to.equal(5);
   });
 
-  it('should not submit empty form', () => {
+  it('does not submit without info', () => {
     const onSubmit = sinon.spy();
-    const form = ReactTestUtils.renderIntoDocument(
+    const form = mount(
       <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
         schema={schema}
-        definitions={definitions}
+        data={{}}
+        formData={{}}
         onSubmit={onSubmit}
         uiSchema={uiSchema}/>
     );
 
-    const formDOM = getFormDOM(form);
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error').length).to.equal(3);
 
-    formDOM.submitForm();
-
-    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(4);
     expect(onSubmit.called).to.be.false;
   });
 
-  it('should submit with valid data', () => {
+  it('submits with required info', () => {
     const onSubmit = sinon.spy();
-    const form = ReactTestUtils.renderIntoDocument(
+    const form = mount(
       <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
         schema={schema}
-        definitions={definitions}
+        data={{}}
+        formData={{}}
         onSubmit={onSubmit}
         uiSchema={uiSchema}/>
     );
-    const formDOM = getFormDOM(form);
+    fillData(form, 'select#root_lastServiceBranch', 'army');
+    fillDate(form, 'root_lastEntryDate', '1990-1-1');
+    fillDate(form, 'root_lastDischargeDate', moment().add(130, 'days').format('YYYY-MM-DD'));
 
-    formDOM.fillData('#root_lastServiceBranch', 'army');
-    formDOM.fillDate('root_lastEntryDate', '2010-1-1');
-    formDOM.fillDate('root_lastDischargeDate', '2011-1-1');
-    formDOM.fillData('#root_dischargeType', 'honorable');
-
-    formDOM.submitForm();
-    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(0);
+    form.find('form').simulate('submit');
+    expect(form.find('.usa-input-error').length).to.equal(0);
     expect(onSubmit.called).to.be.true;
   });
 
-  it('should hide and not require discharge status if discharge date is in the future', () => {
+  it('shows discharge type with lastDischargeDate is in the present or past', () => {
```

Comment:
This test (additional field) and the previous test (basic submission) have been re-arranged now that discharge type is hidden by default, in order to be more consistent with how we typically add tests for additional fields at the end.